3p-ecosystem/src/main/java/com/google/homesampleapp/chip/ClustersHelper.kt

```diff
@@ -204,6 +206,7 @@ class ClustersHelper @Inject constructor(private val chipClient: ChipClient) {
                 override fun onSuccess(values: MutableList<Long>) {
                   continuation.resume(values)
                 }
+
```

Comment:
were these changed by auto generated code?

Reply:
It is our Kotlin formatter (ktfmt).